pages/report-hazard/index.html

```diff
@@ -1,224 +1,298 @@
-              <button>Edit</button>
-            </div>
-          </div>
-        </div>
-
-        <button id="saveReportBtn" type="submit" class="submit">
-          Submit Report
-        </button>
-      </section>
-    </form>
-  </main>
-</body>
+  
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+		<!-- WPA -->
+		<link rel="manifest" href="../../manifest.json" />
+		<link
+			rel="apple-touch-icon"
+			href="../../assets/css/img/icons/icon-96x96.png"
+		/>
+		<meta name="apple-mobile-web-app-status-bar" content="#000000" />
+		<meta name="theme-color" content="#000000" />
+
+		<!-- leaflet CSS for maps -->
+		<link rel="stylesheet" href="../../modules/leaflet/leaflet.css" />
+		<!-- leaflet JS for maps -->
+		<script src="../../modules/leaflet/leaflet.js"></script>
+		<!-- Bootstrap JS -->
+		<script src="../../modules/bootstrap/js/bootstrap.min.js"></script>
+		<!-- Bootstrap CSS -->
+		<link
+			rel="stylesheet"
+			href="../../modules/bootstrap/css/bootstrap.min.css"
+		/>
+		<!-- Variables -->
+		<link rel="stylesheet" href="../../assets/css/variables.css" />
+		<!-- Global Styles -->
+		<link rel="stylesheet" href="../../assets/css/global.css" />
+		<!-- Feature Styles -->
+		<link rel="stylesheet" href="./style.css" />
+		<!-- Global JS -->
+		<script type="module" src="../../global.js" defer></script>
+		<!-- Feature JS -->
+		<script type="module" src="./script.js" defer></script>
+
+		<title>Report Hazard</title>
+	</head>
+
+	<body>
+		<nav>
+			<ul>
+				<li>
+					<a title="Select the location" href="#select-location"
+						>Select Location</a
+					>
+				</li>
+
+				<li>
+					<a
+						title="What type of hazard would you like to report?"
+						href="#hazard-category"
+						>Hazard Category</a
+					>
+				</li>
+
+				<li>
+					<a title="What type of wild animal did you spot?" href="#hazard-type"
+						>Hazard Type</a
+					>
+				</li>
+
+				<li>
+					<a
+						title="Would you like to mention any further details?"
+						href="#additional-details"
+						>Additional Details</a
+					>
+				</li>
+
+				<li>
+					<a
+						title="Please upload photos to help others see the reported hazard visually"
+						href="#upload-photos"
+						>Upload Photos</a
+					>
+				</li>
+
+				<li>
+					<a title="Review the report before submitting." href="#review-report"
+						>Review Report</a
+					>
+				</li>
+			</ul>
+		</nav>
+
+		<main>
+			<form id="reportHazardForm">
+				<section id="select-location" class="page">
+					<h1>1. Select the location.</h1>
+
+					<div>
+						<div>
+							<label>Your current location is:</label>
+							<input type="text" id="locationAddressInput" required />
+						</div>
+
+						<p>You can also move the pin on the map to set your location.</p>
+						<div id="map"></div>
+					</div>
+
+					<a href="#hazard-category">Continue</a>
+				</section>
+
+				<section id="hazard-category" class="page">
```

Comment:
```suggestion
				<section id="hazard-category" class="page">
				       <h1>2. What type of hazard would you like to report?</h1>
				       <div id="hazard-category-content" />
				</section>
```

This HTML structure should be created in the javascript with the list of categories, because the list length may vary

Reply:
dine